controller/tests/test_volunteer_controller.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+import controller.volunteer_controller as vc
+from models.volunteer import Volunteer
+from models.camp import Camp
+
+
+class TestVolunteerController(unittest.TestCase):
+
```

Comment:
To be safe, add a method here:
```python
def setUp(self):
    Volunteer.delete_all()
```
So that previously created volunteers won't interfere with the tests